src/formatGeneratedModule.ts

```diff
@@ -18,8 +18,8 @@ export const formatterFactory = (
     ? `import { ${documentType} } from "${relayRuntimeModule}";`
     : "";
   const docTextComment = docText ? "\n/*\n" + docText.trim() + "\n*/\n" : "";
-  let nodeStatement = `const node: ${documentType ||
-    "never"} = ${concreteText};`;
+  let nodeStatement = `const node = ${concreteText} as ${documentType ||
```

Comment:
Do you think it would make sense to wrap the `concreteText` in an extra set of parentheses? Just to ensure there can never be any ambiguity.

```suggestion
  let nodeStatement = `const node = (${concreteText}) as ${documentType ||
```

Reply:
i've got a soft preference to exclude unnecessary parens just because if someone lints their generated files with `no-extra-parens` then it'll have to rewrite the file every time. that said, it's easy enough excluding the folder from linting. just lemme know & i'll accept the change

Reply:
That _is_ a good point. However, I already think people are and should ignore these artefacts from their prettier/linter setup (we do), as there’s no way to make them fit everybody’s preferences. As such, I think we should optimise for building safe artefacts.